README.md

```diff
@@ -14,148 +14,175 @@ With Pixar's USD system emerging as a powerful tool for 3D graphics pipelines an
-- set environment variable BLENDER_EXE to blender.exe you want to use via the command line or system environment settings.
-- run run_blender_with_rpr.cmd
-
-Example:
-
-`set BLENDER_EXE="C:\Program Files\Blender Foundation\Blender 2.90\blender.exe" && run_blender_with_rpr.cmd`
-
+### Coding Conventions
+Aim is to conform to [pep8](https://www.python.org/dev/peps/pep-0008/). 
+At minimum it's 4 spaces for indentation, sources are utf-8, there's `.gitconfig` in the root of the project - please set you editor to use it (for most simplicity). PyCharm default setting are fine and seems that it also picks up `.editorconfig` automatically also, [Tortoise](https://tortoisegit.org/) Merge has a checkbox 'Enable EditorConfig', for Visual Studio there's [EditorConfig extension](https://visualstudiogallery.msdn.microsoft.com/c8bccfe2-650c-4b42-bc5c-845e21f96328).
+
+### Git
+We try to avoid merge commits, the easiest way to do it. This one rejects merges that would result in merge commit:
+```commandline
+> git config [--global] merge.ff only
+```
+Converts pull to do, essentially, fetch&rebase:
+```commandline
+> git config [--global] pull.rebase true
+```
+Also, make more meaningful commits(one commit per feature) the easy way. This will create a single change set from multiple commits coming from `<branch>`:
+```commandline
+> git merge <branch> --squash 
+```
+
+### ThirdParty libraries
+There is ThirdParty repositories included to the project as a submodules. Please update submodules:
+- `deps/USD` https://github.com/PixarAnimationStudios/USD
+- `deps/HdRPR` https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD
+
+All of them are included via SSH protocol. You will need to create and install [SSH keys](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh).
+
+Once SSH keys are installed update/checkout submodules for active branch:
+```commandline
+> git submodule update --init -f --recursive
+```
+
+### Build
+Require `python 3.9` to be set by default.
+
+#### Windows:
+Use Open x64 Native Tools Command Prompt for Visual Studio 2017 Community and run.
+```commandline
+> git clone https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon
+> cd BlenderUSDHydraAddon
+> git submodule update --init --recursive
+> python tools/build.py -all -bin-dir bin -G "Visual Studio 15 2017 Win64"
+```
+
+#### Linux:
+```commandline
+> git clone https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon
+> cd BlenderUSDHydraAddon
+> git submodule update --init --recursive
+> python tools/build.py -all -bin-dir bin
+```
+
+#### Build tool
+You can build project using `tools/build.py` with different flag combinations. It allows you to create a folder with binaries and copy all the necessary files for development to `/libs` folder. Also `tools/build.py` provides a verity of ways to make a project builds:
+- `-all` - builds all binaries 
```

Comment:
and copies libs to folder (includes -libs flag)
and generates zip (includes -addon flag)

Maybe should add that -all equals to -usd -hdrpr -mx -libs -mx-classes -addon

Reply:
Thanks, added more description.